Source/Views/PageView.swift

```diff
@@ -79,24 +79,7 @@ class PageView: UIScrollView {
   // MARK: - Layout
 
   func configureImageView() {
-    guard let image = imageView.image else { return }
-
-    let imageViewSize = imageView.frame.size
-    let imageSize = image.size
-    let realImageViewSize: CGSize
-
-    if imageSize.width / imageSize.height > imageViewSize.width / imageViewSize.height {
```

Comment:
@onmyway133 I believe it's done for rotation. Have you tested demo with your changed both in portrait and landscape?